omf-python/src/geometry.rs

```diff
@@ -17,11 +17,17 @@ pub struct PyPointSet(pub PointSet);
 #[pymethods]
 impl PyPointSet {
     #[getter]
+    /// Origin of the surface relative to the project coordinate reference
+    /// system.
     fn origin<'py>(&self, py: Python<'py>) -> Bound<'py, PyArray1<f64>> {
         PyArray1::from_slice_bound(py, &self.0.origin)
     }
 
     #[getter]
+    /// Array with `Vertex` type storing the vertex locations.
+    ///
+    /// Add `origin` and the project origin to get the locations relative
+    /// to the project coordinate reference system.
```

Comment:
Maybe for this comment 'These are the positions of the vertices relative to the element origin.' might be enough?

Reply:
IMO change 'Array with `Vertex` type storing the vertex locations' to 'Array with `Vertex` type storing the vertices'.